onnx/backend/test/case/node/attributehasvalue.py

```diff
@@ -23,7 +26,7 @@ def test_one_attribute(name, **kwargs):
                 node,
                 inputs=[],
                 outputs=[output],
-                name="test_attribute_has_value_{name}_false".format(name=name),
+                name=f"test_attribute_has_value_{name}_false",
```

Comment:
May be change to `test_attribute_has_{name}_false` ? Or, omit the `value_` part in the call-site. Otherwise, we will get `value_value_int`.